client/src/models/Address.ts

```diff
@@ -7,6 +7,13 @@ export interface Address {
     apartment?: string;
 }
 
+export interface DBAddress {
```

Comment:
Whats the different between DBAddress and Address?

Reply:
סנדי שינתה, זה 2 מודלז שונים אחד לצרכים שלה עם גוגל ואחד לשלי ושל מניב